agent/scripts/compute-activity-scores.js

```
@@ -0,0 +1,154 @@
+// Computes community activity scores for all GSoC orgs
+// Run via GitHub Actions daily
+
+const fs = require('fs');
```

ESLint

[error] 'require' is not defined.

(no-undef)